src/content/plugins/define-plugin.md

```diff
@@ -101,3 +101,15 @@ new webpack.DefinePlugin({
   'SERVICE_URL': JSON.stringify('https://dev.example.com')
 });
 ```
+
+### Rebuild 
+
+Rebuild wehen the schema updates.
+
+```javascript
+new webpack.DefinePlugin({
+  BUILT_AT: webpack.DefinePlugin.runtimeValue(Date.now, [fileDep, fileDep])
```

Comment:
This example should be extended to explain fileDep, e.g. require a file and pass it as an example dependency of the runtime value. Then demonstrate what happens when file dep changes

Reply:
runtimeValue is new to this plugin, its missing its heading and props explanation/types

Reply:
Done